server.js

```diff
@@ -2,9 +2,9 @@ var express = require('express')
 var app = express()
 
```

Comment:
(Error) Line 2: Expected linebreaks to be 'LF' but found 'CRLF'. - linebreak-style